src/pages/RepoPage/CoverageOnboarding/ExampleBlurb/ExampleBlurb.tsx

```diff
@@ -0,0 +1,37 @@
+import A from 'ui/A'
+
+const ExampleBlurb = () => {
+  return (
+    <div data-testid="example-blurb">
+      &#128193; See example{' '}
```

Comment:
also feels like this should be `an example`. what do you think?

Reply:
I can get behind that, will update